s3/TESTING.md

```diff
@@ -46,17 +48,22 @@ To test as source module
 
 ```bash
 module "test_s3_module" {
-    source = "git::https://github.com/variant-inc/lazy-terraform.git//s3?ref=feature/CLOUD-199-TF-module-for-s3"
-    profile = "devops"
-    region = "us-west-2"
-    bucket_name = "navin-ops-11"
-    purpose = "devops"
-    team = "devops"
-    owner = "devops"
-    lazy_api_host = "https://lazy.apps.ops-drivevariant.com"
-    lazy_api_key = "00o2TvfA8DjInc-RuQ1tC6nwqOnsqZ-3bsVyTuBHGK"
-    octopus_project_space = "test-space"
-    octopus_project_name = "test-project"
+    source = "git::https://github.com/variant-inc/lazy-terraform.git//s3?ref=feature/CLOUD-402-add-tags-to-s3"
+        profile = "devops"
+        region = "us-west-2"
+        bucket_name = "navin-ops-11"
+        lazy_api_host = "https://lazy.apps.ops-drivevariant.com"
+        lazy_api_key = "00o2TvfA8DjInc-RuQ1tC6nwqOnsqZ-3bsVyTuBHGK"
+        user_tags = {
+        team = "devops4"
+        purpose = "s3-test3"
+        owner = "naveen3"
+        }
+        octopus_tags = {
+        project = "actions-test3"
+        space   = "Default3"
+        }
+        replication=true
 
 }
 
```

Comment:
Still need to add AWS_DEFAULT_REGION to readme and testing steps

Reply:
Removed